Source/backend/models/Product.js

```diff
@@ -0,0 +1,16 @@
+import mongoose from 'mongoose';
+
+const productSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
```

Comment:
Consider adding validation for the name field to ensure it’s not empty or too short.